src/Education/Layout/Logical/HowToFixItSection.cs

```diff
@@ -45,10 +46,21 @@ public HowToFixItSection(List<HowToFixItSectionContext> contexts)
 
         public string Title => "How can I fix it?";
 
-        public IAbstractVisualizationTreeNode GetVisualizationTreeNode(ITabsRepository tabsRepository)
+        public IAbstractVisualizationTreeNode GetVisualizationTreeNode(IStaticXamlStorage staticXamlStorage)
         {
-            throw new NotImplementedException();
-            // todo in the next PR
+            if (partialXamlContent != null)
```

Comment:
This feels wrong. The class does two different things. But it's very simple so splitting the class could be overoptimization. Have you considered a quick way to do it?

Reply:
I chose this way because it's less boilerplate. I'm not sure if splitting this into two classes would add any value